services-js/access-boston/scripts/post-deploy.ts

```diff
@@ -0,0 +1,80 @@
+// This is a script in the service, rather than a binary in deploy-tools,
+// because binaries from our packages are not available in the container because
+// of the order we do package.json / yarn install in the Dockerfile.
+
+/* eslint no-console: 0 */
+import path from 'path';
+
+import {
+  uploadToS3,
+  parseBranch,
+  uploadSourceMapsToRollbar,
+  reportRollbarDeploy,
+} from '@cityofboston/deploy-tools';
+
+// Important for getting ASSET_HOST and ROLLBAR_ACCESS_TOKEN configuration. This
+// script is run in the container from the default entrypoint, which would have
+// downloaded the config files.
+require('dotenv').config();
+
+const NEXT_SRC_PATH = path.join('build', '.next');
+const NEXT_DST_PATH = '_next';
+
+const { environment, serviceName } = parseBranch(process.env.DEPLOY_BRANCH!);
+const bucketEnvironment = environment === 'production' ? 'prod' : 'staging';
+const bucket = `cob-digital-apps-${bucketEnvironment}-static`;
+
+(async function() {
+  console.error(`🛫  Uploading service ${serviceName} to ${bucket}…`);
+  console.error();
+
+  // Because all of Next’s JS is versioned, we don’t need to separate out these
+  // uploads by variant. We can just push everything and anything that needs
+  // separate file names will have it.
+  await uploadToS3(
+    NEXT_SRC_PATH,
+    bucket,
+    `${serviceName}/${NEXT_DST_PATH}`,
+    // 1 year cache expiration because all of Next’s JS is versioned
+    60 * 60 * 24 * 365
+  );
+
+  console.error();
+
+  console.error(`💅 Successfully uploaded ${serviceName} to ${bucket}.`);
```

Comment:
It never occurred to me how perfect this emoji is to express ”successful action”...